includes/Core/Storage/HasConnectedAdmins.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Storage\HasConnectedAdmins
+ *
+ * @package   Google\Site_Kit\Core\Storage
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Storage;
+
+use Google\Site_Kit\Core\Authentication\Clients\OAuth_Client;
+use Google\Site_Kit\Core\Storage\Options_Interface;
+use Google\Site_Kit\Core\Storage\Setting;
+use Google\Site_Kit\Core\Storage\User_Options_Interface;
+
+/**
+ * ConnectedAdmins class.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class HasConnectedAdmins extends Setting {
```

Comment:
- Please add a test case for this class.
- Please implement the `\Google\Site_Kit\Core\Storage\Setting::get_type` as `boolean`

Reply:
Updated.